test/l3-resources/atlas-bootstrap/README.md

```diff
@@ -0,0 +1,22 @@
+# @awscdk-resources-mongodbatlas/atlas-basic
+
+## CFN resource type used
+
+- AWS::IAM::Role
+- AWS::IAM::Policy
+- AWS::CloudFormation::TypeActivation
+- AWS::SecretsManager::Secret
+
+These CFN resources must be active in your AWS account while using this constructor.
```

Comment:
I didn't get it.

All the `AWS::*` resources are built-in and don't need any activation. Why should we activate them?

Reply:
i found that AWS::CloudFormation::TypeActivation is not acivated for some reason in a few regions, this comment is just for the developers, to know which resources should be activated if thew want to test, but dont worry about this one, is just for developer testing, the user should not use this